docs/API.md

```diff
@@ -407,28 +662,42 @@ await realm.invitePlayer('a8005260a332457097a50bdbe48a9a21', 'Steve')
 
 #### open
 
-() => Promise\<void>
+() => Promise\<boolean>
 
 Opens a Realm. Allows players to join
 
 ```js
 await realm.open()
 ```
 
-No output
+True if the world has opened
```

Comment:
Can't this be an exception? Otherwise example code needs to handle the false case.

Reply:
There is no false case, as stated [here](https://wiki.vg/Bedrock_Realms#PUT_.2Fworlds.2F.7Bid.7D.2Fopen) it doesn't return false, it returns the retry again later string. Should we be returning false from this function considering it's basically false if it says retry again later.

Reply:
I also understand this too, sorry about that.